src/monodroid/CMakeLists.txt

```diff
@@ -56,22 +64,35 @@ endif()
 
 xa_common_prepare()
 # Don't put the leading '-' in options
-set(TEST_COMPILER_ARGS ${XA_COMPILER_FLAGS}
+set(TEST_COMPILER_ARGS_CPP ${XA_COMPILER_FLAGS}
   fno-rtti
   fno-exceptions
+  Wall
```

Comment:
Don't we likewise want all of these warnings in `TEST_COMPILER_ARGS_C`?

Reply:
No, because our only C code is 3rd party too - the `zip/*` files (which I hope to replace with "real" zlib at some point)